code/modules/mob/living/carbon/human/species.dm

```diff
@@ -795,6 +795,11 @@
 
 			if(istype(J) && J.allow_thrust(0.01, H))
 				. -= 2
+			else
+				var/obj/item/organ/internal/cyberimp/chest/thrusters/T = H.getorganslot("thrusters")
+				if(istype(T) && T.allow_thrust(0.01, H))
+					. -= 2
+
```

Comment:
This needs to be in carbon since monkeys and xenomorphs(some really ballsy human doing this surgery) could get this implant.